api/src/damnit_api/filewatcher/mtime_cache.py

```diff
@@ -0,0 +1,26 @@
+import os
+import time
+from pathlib import Path
+TTL = 5.0
+
+file_mtime_cache: dict[str, dict[str, float]] = {}
+
+
+def get(file_path: str) -> float:
```

Comment:
Agreed with Thomas: could you please provide a more descriptive name here?

Reply:
It's also good that you implemented your own TTL cache :blush: maybe you could consider using a library to help make the code cleaner? 

https://github.com/tkem/cachetools/

Reply:
Maybe, I have no strong opinions about it. Though we use `alru_cache` for async caches with TTLs.
https://github.com/search?q=repo%3AEuropean-XFEL%2FDAMNIT-web%20alru_cache&type=code

Reply:
We agreed to leave this implementation for now. It's easy to convert it to use `alru_cache` or similar later.